src/app/containers/MediaPageMain/index.jsx

```diff
@@ -7,15 +7,40 @@ import ATIAnalytics from '../ATIAnalytics';
 import MetadataContainer from '../Metadata';
 import { Grid, GridItemConstrainedMedium } from '../../lib/styledGrid';
 import { ServiceContext } from '../../contexts/ServiceContext';
+import aresBlockPropTypes from '../../models/propTypes/mediaPage/content/aresBlock';
+
+const renderBlock = ({ block, script, service, key }) => {
+  if (block.type === 'heading') {
+    return (
+      <Headline script={script} service={service} key={key}>
```

Comment:
Rather than calling them like this, can you call them like this?
```
const renderBlock = ({ block, script, service, key }) => {
  const Component = {
    heading: Headline,
    paragraph: Paragraph,
  }[block.type];

  if (!Component) {
    return null;
  }

  return (
    <Component script={script} service={service} key={key}>
      {block.text}
    </Component>
  );
};
```

Reply:
Yeah can do, this was only temporary until we need to extend it for images and other features on the MAP page.